Apromore-Custom-Plugins/SplitMiner-Logic/src/main/java/org/apromore/splitminer/SplitMiner.java

```diff
@@ -187,12 +187,12 @@ public BPMNDiagram discoverFromDFGP(DirectlyFollowGraphPlus idfgp) throws Except
             e.printStackTrace();
             try{
             	dfgp.resetDFGPStructures();
-                dfgp.buildSafeDFGP(); //recreate a safe data structure for the graph in case errors happen (the graph could be disconnected)  
+                dfgp.buildSafeDFGP(); //recreate a safe data structure for the graph in case errors happen (the graph could be disconnected)
                 transformDFGPintoBPMN();
                 if (structuringTime == SplitMinerUIResult.StructuringTime.POST) structure();
             } catch ( Exception ee ) {
                 System.out.println("ERROR - something went wrong in translating DFG to BPMN");
-                e.printStackTrace();            
+                e.printStackTrace();
```

Comment:
Nothing you changed, but this is logging to stdout rather than using the system logger.